cowdao_cowpy/order_book/api.py

```diff
@@ -82,6 +84,29 @@ async def get_order_by_uid(
             response_model=Order,
         )
 
+    async def get_order_multi_env(
+        self, order_uid: UID, context_override: Context = {}
+    ) -> Order | None:
+        for env in ENVS_LIST:
+            # TODO extract & exclude current env from loop.
+            try:
+                # TODO: context override does not appear to work as expected.
+                result = await self.get_order_by_uid(
+                    order_uid, {**context_override, "env": env.value}
+                )
+                return result
+            except UnexpectedResponseError:
+                pass
+
+    async def get_order_competition_status(
```

Comment:
think we have a duplicate function now! I think

https://github.com/cowdao-grants/cow-py/blob/fb15c5218a723d4f4e0ff420c39243d9b9c2832e/cowdao_cowpy/order_book/api.py#L208